newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -590,12 +602,11 @@ const SpritesList = ({
           resourcesLoader={resourcesLoader}
           direction={direction}
           project={project}
-          onSortEnd={onSortEnd}
-          onAddSprite={onAddSprite}

```

Comment:
This was useless?

Reply:
Indeed, the SortableList, based on the SortableContainer, do not use this prop. It's typed as React.component<any> so hard to spot. (I tried improving the typing, but spent too much time without a clean solution. The library is in TS, need to create our own flow typings etc...)